web/libs/editor/src/components/ImageView/ImageView.jsx

```diff
@@ -583,12 +587,14 @@ export default observer(
       // shape we can click on. Here we're relying on cursor position and non-transparent pixels
       // of the mask to detect cursor-region collision.
       if (ff.isActive(FF_BITMASK)) {
-        const selectedRegion = item.selectedRegions.find((r) => r.type === "bitmaskregion");
-        const currentTool = item.getToolsManager().findSelectedTool().toolName;
+        const hasSelected = item.selectedRegions.some((r) => r.type === "bitmaskregion");
+        const isBitmask = ["BitmaskTool", "BitmaskEraserTool"].includes(
+          item.getToolsManager().findSelectedTool().toolName,
+        );
 
         // We want to avoid weird behavior here with drawing while selecting another region
         // so we just do nothing when clicked outside AND we have a tool selected
-        if (selectedRegion && currentTool === "BitmaskTool") {
+        if (hasSelected && currentTool === "BitmaskTool") {
```

Comment:
`currentTool` is removed already in favor of simple boolean `isBitmask`, so let's fix the condition
```suggestion
        if (hasSelected && isBitmask) {
```